src/test/groovy/net/rubygrapefruit/platform/file/AbstractFileEventsTest.groovy

```diff
@@ -280,6 +280,21 @@ abstract class AbstractFileEventsTest extends Specification {
         expectedChanges.await()
     }
 
+    def "does not receive events after directory is unwatched"() {
```

Comment:
There are tests missing for what happens when
- `startWatching` twice for the same path
- `stopWatching` for a non-watched path
- `stopWatching` twice for the same watched path.

Reply:
Good point, adding.